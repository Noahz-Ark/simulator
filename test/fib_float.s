../../compiler/mytest/fib_float.s