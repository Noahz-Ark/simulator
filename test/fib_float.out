832040.000000